packages/config-utils/proxy.js

```diff
@@ -53,8 +57,10 @@ module.exports = ({
 
   let agent;
 
-  if (env.startsWith('prod') || env.startsWith('stage')) {
-    // PROD and stage are deployed with Akamai which requires a corporate proxy
+  const isProd = env.startsWith('prod');
+
+  if (env.startsWith('stage') || useAgent) {
+    // stage is deployed with Akamai which requires a corporate proxy
```

Comment:
We need the proxy agent in prod as well. The `useAgent` should be `true` by default.

Reply:
I do not think so because now PROD works without the agent thanks to the bouncing. So by default it will work for anyone un/connected to VPN. But this option is made for people who are connected to the VPN and for whatever reason they do not want to use the bouncing. (There could and probably there would be some hidden bugs.)

Reply:
Yeah, Patch team wanted to develop on prod without the vpn. So I tried to fix that... and I came up with the bounce mechanism included in this PR. Check the bypass function.

Reply:
Can we first hide it entirely behind a flag? So we can test a bit before updating every config? I would like to avoid breaking any other local development setup.

I have a few services I can test it that I know do not work in prod env. (One of which is chrome)

Reply:
Sure, I can switch the default variables to true<>false, I just thought the default behavior without VPN makes more sense.